IdokladSdk/Requests/Core/BaseList.cs

```diff
@@ -179,7 +177,7 @@ protected ApiResult<Page<TResult>> ApplySelectorFunction<TResult>(ApiResult<Page
             {
                 Data = new Page<TResult>
                 {
-                    Items = apiResult.Data?.Items?.Select(o => selectorFunction.Invoke(o)) ?? null,
+                    Items = apiResult.Data?.Items?.Select(o => selectorFunction.Invoke(o)),
```

Comment:
?? null jsi smazal, protože pokud jsou Data nebo Items == null, pak výsledkem je stejně null a "?? null" bylo zbytečné?

Reply:
áno, presne preto som to zmazal, bolo to zbytočné